### ISSUE TITLE
Jellyfin chromecast client unresponsive to any voice commands.

### ISSUE THREAD

**Issue by bengalih** [jellyfin/jellyfin-chromecast#29]
When streaming from Plex to my Google Mini (Nest), I am able to say to "Ok, Google.  stop playing"  or "OK, Google.  cancel" and The stream will terminate.  Similarly, "Ok, Google.  Pause/Resume" works.

When using Jellyfin, this doesn't work (and never has).  The stream pauses long enough for Google to listen and think for a moment, but then the stream continues to play.  If you then ask "Ok Google, what's playing?"  it will say nothing is playing.

In short, the JF chromecast client doesn't seem to have any intelligence at all to be able to actually interact with google.  The only way to control it is from a cast client (i.e. JF app on web or mobile).
This is really inconvenient when needed to stop a playing stream immediately (like a phone call, etc.).

**Comment by YouKnowBlom** [comment]
This will be fixed in the large update/rewrite coming out soon™

**Comment by alexdelorenzo** [comment]
> This will be fixed in the large update/rewrite coming out soon™

Hey, how is this coming along? I ask because I tried to test out a nightly build, but couldn't get casting to work, and I'd like to help if there is still work to be done to get this fixed.

**Comment by ferferga** [comment]
@alexdelorenzo Did you switch to the new Chromecast version in your user's settings? See jellyfin/jellyfin-web#1242 for reference

We don't switch by default to the new Chromecast client because it still needs more testing.

**Comment by MrTimscampi** [comment]
Do note that, currently, the unstable Chromecast option in the nightly still gives you a copy of the stable one.

That's fixed soon, I believe (@dkanada can tell you more)

**Comment by dkanada** [comment]
I'm working on the continuous delivery to update the nightly with every merge to this repository.

**Comment by YouKnowBlom** [comment]
Closing this as it's a duplicate of #67
